app/scripts/modules/kubernetes/src/v2/pipelines/stages/runJob/KubernetesV2RunJobStageConfig.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+
+import {
+  IStageConfigProps,
+  AccountService,
+  //   HelpField,
```

Comment:
Remove? Unless you think a help field might be helpful on the manifest entry field!